cmd/local/uninstall.go

```diff
@@ -28,12 +28,25 @@ func NewUninstallCmd() *cobra.Command {
 				return err
 			}
 
+			err = k3d.UninstallRegister()
```

Comment:
Do we always want to uninstall the register or only when we purge / deleteregister flags are true?

Reply:
yes